tests/test_pandas_coverage.py

```diff
@@ -75,19 +84,28 @@ def test_rename(global_var):
 
     assert df.data_type_lookup["Name"] == new_df.data_type_lookup["Car Name"]
 
-    assert df.data_type != new_df.data_type
+    data_type = df.compute_data_type_from_lookup()
+    new_data_type = new_df.compute_data_type_from_lookup()
+
+    assert data_type != new_data_type
+
+    assert data_type["nominal"][0] == "Name"
+    assert new_data_type["nominal"][0] == "Car Name"
+
+    data_model_lookup = df.compute_data_model_lookup()
+    new_data_model_lookup = new_df.compute_data_model_lookup()
 
-    assert df.data_type["nominal"][0] == "Name"
-    assert new_df.data_type["nominal"][0] == "Car Name"
+    assert data_model_lookup != new_data_model_lookup
 
-    assert df.data_model_lookup != new_df.data_model_lookup
+    assert data_model_lookup["Name"] == new_data_model_lookup["Car Name"]
```

Comment:
Here maybe we could just compute the equality between the `data_model`s

Reply:
So, do you want me to get rid of lines 98-100?